### ISSUE TITLE
dependencies clash on tsutils

### ISSUE THREAD

**Issue by samiq** [ajafff/tslint-consistent-codestyle#98]
there seems to be a dependencies clash in npm between this package and one from fimbul/ymir which requires tsutils@>2.29.0, while this package is requiring 2.27.2. 

while installing it throws a warning:

```
npm WARN @fimbul/ymir@0.15.0 requires a peer of tsutils@>2.29.0 but none is installed. You must install peer dependencies yourself.
```
would be great to know how to not get this warning

**Comment by ajafff** [comment]
This is not really an issue as your package manager should install the latest matching version of `tsutils`. Maybe tsutils is locked to an older version in your package-lock.json

Anyways, I'll publish a fix in a few minutes
